third_party/move/tools/move-coverage/src/source_coverage.rs

```diff
@@ -27,15 +27,23 @@ use std::{
     str::FromStr,
 };
 
+/// Function source coverage.
 #[derive(Clone, Debug, Serialize)]
 pub struct FunctionSourceCoverage {
+    /// Indication for native functions.
     pub fn_is_native: bool,
+
+    /// List of uncovered locations in the function.
```

Comment:
```suggestion
    /// List of source locations in the function that were not covered.
```

Reply:
I haven't looked at the code deeply, I am assuming this is what it means from looking here. Please double check.

Reply:
Yes, that's right! I'm using this in the `move-mutation-test` a lot these days.
Done